test/unit/range/views/view_kmer_hash_test.cpp

```diff
@@ -206,3 +206,22 @@ TYPED_TEST(kmer_hash_ungapped_test, issue1719)
         EXPECT_EQ(4u, v3.size());
     }
 }
+
+//https://github.com/seqan/seqan3/issues/1754
+TYPED_TEST(kmer_hash_ungapped_test, issue1754)
+{
+    TypeParam text1{'A'_dna4, 'C'_dna4, 'G'_dna4, 'T'_dna4, 'A'_dna4, 'G'_dna4, 'C'_dna4}; // ACGTAGC
```

Comment:
Does `"ACGTAGC"_dna4` not suffice here?

Reply:
No, because in the list of the TypeParam is also std::list and that requires this not so nice writing of the text. We could test only on a dna4_vector and write it with `"ACGTAGC"_dna4`, but then we test less...